src/navigation/navigation.cc

```diff
@@ -116,6 +116,25 @@ bool isBetween(const Vector2f p0, const Vector2f p1, const Vector2f p2, const Ve
 	}
 }
 
+// whoops, this never actually gets used ¯\_(ツ)_/¯
+float getAngleBetween(const Vector2f point_A, const Vector2f point_B, const Vector2f point_C){
+	/* returns absolute value of angle between AC and BC
+	          C
+			/   \
+		   / thC \
+		  a       b
+		 /         \
+		/           \
+	   B______c______A
+	*/
+	float side_a = (point_B-point_C).norm();
+	float side_b = (point_A-point_C).norm();
+	float side_c = (point_A-point_B).norm();
+	// Law of Cosines
+	float angle_C = acos((side_c*side_c-side_a*side_a-side_b*side_b)/(-2*side_a*side_b));
+	return angle_C;
+}
+
```

Comment:
This could be done more efficiently with a dot product calculation, rather than taking 3 norms

Reply:
Two norms and a dot product is not too much different computationally, but also I didn't end up using this function so do whatever you want with it :)